wgsl/index.bs

```diff
@@ -9397,11 +9431,20 @@ Each structure member in the entry point IO [=shader-creation error|must=] be on
 (see [[#builtin-inputs-outputs]]), or assigned a location.
 
 Locations [=shader-creation error|must not=] overlap within each of the following sets:
-* Members within a structure type.
+* Members within a structure type for a given [=attribute/blend_src=] value (a single set if none is specified).
```

Comment:
I don't know what "a single set" means here

Reply:
@kainino0x I've changed the sentence here. PTAL, thanks!